### DISCUSSION TITLE
socks代理无法正常访问

### DISCUSSION THREAD

**Discussion by ttxs25830** [v2fly/v2ray-core#1729] [Q&A] [answered]
Server主机运行在墙外，client运行在本地内网192.168.1.2上，我从内网链接代理
测试发现使用Proxy SwitchyOmega可以正常链接代理访问外网，但是使用Windows设置的代理（确认设置为socks类型）或是火狐设置代理无法访问，确认时间配置无问题，确认防火墙正常开放端口。
[Downloads.zip](https://github.com/v2fly/v2ray-core/files/8490002/Downloads.zip)
以上是两个配置文件以及通过火狐访问https://www.google.com/ 时的error.log

**Comment by AkinoKaede** [comment]


**Reply by ttxs25830** [reply]
So, if I do not set its vision, it will use socks5 by default?

**Comment by ttxs25830** [comment, marked as answer]
已自行解决，似乎与socks协议无关，问题在于服务端与客户端之间通信
